test/xcribe/writter_test.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Xcribe.WritterTest do
+  use ExUnit.Case, async: false
+
+  import ExUnit.CaptureIO
+
+  alias Xcribe.Writter
+
+  @output_path "/tmp/test"
```

Comment:
`test` is a very common name and it's possible already has a file with this name. Perhaps, to validate the file has been written in all test runs we should generate a random file name.

```elixir
output_path = "/tmp/xcribe_#{:rand.uniform()}"